packages/admin-ui/src/lib/marketing/src/components/promotion-list/promotion-list.component.ts

```diff
@@ -73,11 +73,17 @@ export class PromotionListComponent
                 filterField: 'description',
             },
             {
-                name: 'usageLimit',
+                name: 'perCustomerUsageLimit',
                 type: { kind: 'number' },
                 label: _('marketing.per-customer-limit'),
                 filterField: 'perCustomerUsageLimit',
             },
+            {
+                name: 'usageLimit',
+                type: { kind: 'number' },
+                label: _('marketing.limit'),
```

Comment:
The label here should match the translation token used in the detail form so that only 1 translation is created. I'd say `marketing.usage-limit` is better because it is more explicit.